apps/vr-tests-react-components/src/stories/Card.stories.tsx

```diff
@@ -60,43 +60,7 @@ storiesOf('Card Converged', module)
         />
       </Card>
     </div>
-  ))
-  .addStory(
-    'appearance',
-    () => (
-      <div style={{ display: 'flex', flexDirection: 'column', gap: '16px' }}>
-        <div>
-          <h1>Filled</h1>
-          <Card appearance="filled">
-            <SampleCardContent />
-          </Card>
-        </div>
-        <div>
-          <h1>Filled alternative</h1>
-          <Card appearance="filled-alternative">
-            <SampleCardContent />
-          </Card>
-        </div>
-        <div>
-          <h1>Outline</h1>
-          <Card appearance="outline">
-            <SampleCardContent />
-          </Card>
-        </div>
-        <div>
-          <h1>Subtle</h1>
-          <Card appearance="subtle">
-            <SampleCardContent />
-          </Card>
-        </div>
-      </div>
-    ),
```

Comment:
These stories use the `<h1>` tag, which was ending up with the wrong line height after this change (bug in the test). It could have been replaced by `<Title1 as="h1">`; however, this is all entirely redundant with the "appearance interactive - Filled", etc. stories below, which are testing the same appearances. So I deleted them.

Reply:
This is NOT redundant. Filled and Interactive Filled use different styles, see https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-card/src/components/Card/useCardStyles.ts#L54.
Please add the test back and either change the headers or let the visual regression test fail, which means it's doing it's job correctly, and we can approve as it's intended.

Reply:
Ok I'll revert these changes, thanks!